process/block/metablock.go

```diff
@@ -786,7 +761,7 @@ func (mp *metaProcessor) CreateBlock(
 	return metaHdr, body, nil
 }
 
-func (mp *metaProcessor) isPreviousBlockEpochStart() (uint32, bool) {
+func (mp *baseProcessor) isPreviousBlockEpochStart() (uint32, bool) {
```

Comment:
move this to baseprocessor if you renamed the pointer structure.

Reply:
Right, missed it, fixed.